modules/oe_whitelabel_helper/oe_whitelabel_helper.services.yml

```diff
@@ -4,3 +4,5 @@ services:
     arguments: ['@plugin.manager.block']
     tags:
       - { name: twig.extension }
+  oe_whitelabel_helper.slim_select:
```

Comment:
A few thoughts about this:
-  services should be reusable classes (as in can be used anywhere), this class is specific for the requirements alter and the info alter, so it is not suited for a service.
- we dont want this to be used as like an api so it should have been marked with `public: false` but that brings a whole different set of problems when trying to use in a preprocess, so I think we should drop the service part.

We still want code reuse and OOP so I thought more along the lines of a helper class in oe_whitelabel/src (see we have 2 more there), And then we call the class with \Drupal::classResolver(SlimSelectHelper::class);

Reply:
oky good idea, it's done